src/test/java/com/microsoft/azure/relay/SendReceiveTest.java

```diff
@@ -240,6 +273,32 @@ public void httpWriteResponseEnsureOrderTest() throws IOException {
 		httpRequestSender("POST", String.join("", messages), smallStr);
 	}
 	
+	@Test
+	public void httpWriteReceiveAllByteValuesTest() throws IOException {
+		byte[] byteSet = getByteSet();
+		listener.setRequestHandler(context -> {
+			RelayedHttpListenerResponse response = context.getResponse();
+			response.setStatusCode(STATUS_CODE);
```

Comment:
>setStatusCode [](start = 12, length = 13)

Are there any tests in this suite which use any other status code?

Reply:
Do you feel like there should be separate tests that just focus on the status code and description? I didn't include any because I didn't think there are any special values that we need to test specifically

---
In reply to: [294381396](https://github.com/Azure/azure-relay-java/pull/47#discussion_r294381396) [](ancestors = 294381396)

Reply:
I'd feel better the tests do at least 1 variation (2 total) on StatusCode. Otherwise how do you know it's actually using the one you told it to? Same thing with Description.

---
In reply to: [295535676](https://github.com/Azure/azure-relay-java/pull/47#discussion_r295535676) [](ancestors = 295535676,294381396)

Reply:
That makes sense, I will try to take that into account probably in a future PR.

---
In reply to: [295557975](https://github.com/Azure/azure-relay-java/pull/47#discussion_r295557975) [](ancestors = 295557975,295535676,294381396)